templates/default/systemd/alertmanager.service.erb

```diff
@@ -6,9 +6,11 @@ After=network.target
 ExecStart=<%= node['prometheus']['alertmanager']['binary'] %> \
     -log.level=debug \
     -storage.path="<%= node['prometheus']['dir'] %>/data" \
-    -config.file=/opt/prometheus/alertmanager.conf \
-    -web.external-url=http://127.0.0.1/alert-manager/
-User=prometheus
+    -config.file="<%= node['prometheus']['alertmanager']['config.file'] %>" \
+    -web.external-url="<%= node['prometheus']['flags']['alertmanager.url'] %>"
```

Comment:
I'll look into adding some tests for this. I didn't see *any* tests for Alertmanager in this repo so I might go ahead and just do some smoke tests for the whole thing while I'm at it.